pkg/ruler/ruleset.go

```diff
@@ -298,10 +299,13 @@ func GenerateInTotoLink(reports []Report, fileBytes []byte) in_toto.Metablock {
 
 	linkMb.Signatures = []in_toto.Signature{}
 	linkMb.Signed = in_toto.Link{
-		Type:      "link",
-		Name:      "kubesec",
-		Materials: materials,
-		Products:  products,
+		Type:        "link",
+		Name:        "kubesec",
+		Materials:   materials,
+		Products:    products,
+		ByProducts:  map[string]interface{}{},
+		Command:     []string{},
+		Environment: map[string]interface{}{},
```

Comment:
Should these be left empty or be set to something?
e.g. `Command: ["kubesec","scan","somefile.yaml"]` ?

What happens in the case of running as a server & submitting over API? `kubesec http 8080`

Reply:
Yeah sounds good.
Could you mark it as a TODO?